datadog/dogstatsd/base.py

```diff
@@ -32,6 +33,11 @@
 # Tag name of entity_id
 ENTITY_ID_TAG_NAME = "dd.internal.entity_id"
 
+# Default buffer settings
+UDP_OPTIMAL_PAYLOAD_LENGTH = 1432
+UDS_OPTIMAL_PAYLOAD_LENGTH = 8192
+MAX_BUFFER_ITEM = sys.maxsize
```

Comment:
`sys.maxsize` is way to big to be used as a default value. Also what the point of still using `MAX_BUFFER_ITEM` since we now computing the actual byte size ?

I think we should drop the feature of setting a manual limit on the number of metrics per packet (it's arbitrary and never work properly).

Reply:
It was mostly for consistency purpose between clients (https://github.com/DataDog/datadog-go/blob/master/statsd/options.go#L17). I agree that settings an arbitrary limit is not a good idea and I don't mind dropping the feature.